pages/common/nc.md

```diff
@@ -4,7 +4,7 @@
 
 - Listen on a specified port:
 
-`nc  -l {{port}}`
+`nc  -l -p {{port}}`
```

Comment:
Actually the man pages say something else. As far as I understand -

`-p` is used to specify the source port. And the port argument is the destination port. 

For use along side the `-l` flag, this is what the man page says -
```
-l      Used to specify that nc should listen for an incoming connection
             rather than initiate a connection to a remote host.  It is an
             error to use this option in conjunction with the -p, -s, or -z
             options.
```

Can you check again from your end why it does not work without the `-p` flag ?